### DISCUSSION TITLE
install in laravel 8

### DISCUSSION THREAD

**Discussion by abbasfisal** [vuetifyjs/vuetify#13110] [Q&A]
Is there an easier way to install and use **Vueitify** in Laravel?

**Comment by KaelWD** [comment]
Use laravel just as an API server and build your frontend with vue-cli

**Reply by abbasfisal** [reply]
Could u explain it more? is there any doc or tut(vids) ?

**Reply by jamaluddinrumi** [reply]
i've successfully installed vuetify in laravel 8 jetstream with inertia stack 

here it is https://github.com/jamaluddinrumi/inutrition 

you are welcome to clone it

**Reply by abbasfisal** [reply]
tnx

**Reply by gcavanunez** [reply]
It's essentially a requirement to use postcss 7 and mix version 5 right?
